### PR TITLE
added grid component

### PR DESCRIPTION
**Pull request by tusharkandpal**
To check:
✅ added grid component
✅ added respective component color in `doc.css` & other style changes.
✅ added their respective links in other html files.

- created rating, text-utilities & toast component

### CODE REVIEW

**Review by harshalini — APPROVED** [review]
Good naming conventions, good to merge

File: components/grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+          <p class="component-text mpy-s">
+            This layout renders the 20:80 ratio parent containers.
+          </p>
+          <div class="component-display mp-m">
+            <div class="grid grid-20-80 gap">
+              <div style="border: 2px dashed var(--secondary-color)">
+                Grid I
+              </div>
+              <div style="border: 2px dashed var(--secondary-color)">
+                Grid II
```
Comment (sushil-mod) [inline components/grid/grid.html:107]: if possible ,you can remove inline style same in line 95

File: components/grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+          <h3 class="component-header font-l">Grid 50-50 Layout</h3>
+          <p class="component-text mpy-s">
+            This layout renders the 50:50 ratio parent containers.
+          </p>
+          <div class="component-display mp-m">
+            <div class="grid grid-50-50 gap-y">
+              <div style="border: 2px dashed var(--secondary-color)">
+                Grid I
+              </div>
+              <div style="border: 2px dashed var(--secondary-color)">
```
Comment (sushil-mod) [inline components/grid/grid.html:137]: here also

**Review by sushil-mod — APPROVED** [review]
have some inline styles ,rest is good code

File: components/grid/grid.html (hunk 1 above)
Comment (tusharkandpal) [reply components/grid/grid.html:107]: Yeah that is because i want user could remove it. It shouldn't be part of my library.

**Pull request merged**